components/Footer/Footer.js

```diff
@@ -22,27 +25,33 @@ const NavList = styled.ul`
 `;
 
 const NavItem = styled.li`
-  margin: 0 10px;
+  display: flex;
+  margin: 0 0.3rem;
+  align-items: center;
+  color: #cdcdcd;
 `;
 
 const NavLink = styled(Link)`
-  color: black;
+  color: #cdcdcd;
   text-decoration: none;
   font-size: 1rem;
-  font-weight: bold;
+  padding-right: 0.2rem;
 `;
 
 export default function Footer() {
   return (
     <FooterContainer>
       <NavList>
         <NavItem>
+          <FiLayout style={{ marginRight: "0.2rem" }} />
```

Comment:
This sounds duplicated, what about a css variable to set the margin for all three links at once?